vignettes/dwelling_type_bar_graph.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "Bar Graph Demo"
+author: "Jens von Bergmann"
+date: "2017-08-15"
+output: html_notebook
+#output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+We demonstrate how to import data from the [CensusMapper](https://CensusMapper.ca) API. The code for this notbook can be found on [GitHub](https://github.com/mountainMath/cancensus).
+
+This example demonstrates how to dynamically discover regions and vectors and produce an overview graph based on the data by using the `cancensus.list_regions`, `cancensus.list_vectors` and `cancensus.search_vectors` methods.
+
+```{r API Key, echo=TRUE, warning=FALSE}
+library(cancensus)
+# set your API key
+# options(cancensus.api_key)='<your API key>'
+
+# use 2016 data
+dataset='CA16'
+```
+
+### Dynamically Select Regions
+We select the top 6 CMAs by population.
+```{r, echo=TRUE, message=FALSE, warning=FALSE}
+# grab 6 largest CMAs
+cma_list <- cancensus.list_regions(dataset) %>% 
+  filter(level=='CMA') %>% 
+  top_n(6, pop) %>%
+  pull(region)
+regions=list(CMA=cma_list)
+```
```

Comment:
Yes, skip the `pull(region)` line and replace the `regions=list....` with the `as_region_list(cma_list)`.